Ghidra/Processors/V850/data/languages/V850.cspec

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<compiler_spec>
+
+  <global>
+    <range space="ram"/>
+  </global>
+
+  <stackpointer register="sp" space="ram"/>
+
+  <default_proto>
+    <prototype name="__stdcall" extrapop="0" stackshift="0">
+      
+      <input>
+        <pentry minsize="1" maxsize="4">
+          <register name="r6"/>
+        </pentry>
+
+        <pentry minsize="1" maxsize="4">
+          <register name="r7"/>
+        </pentry>
+
+        <pentry minsize="1" maxsize="4">
+          <register name="r8"/>
+        </pentry>
+
+        <pentry minsize="1" maxsize="4">
+          <register name="r9"/>
+        </pentry>
+
+        <pentry minsize="1" maxsize="500" align="4">
+          <addr offset="0" space="stack"/>
+        </pentry>
+      </input>
+      
+      <output>
+        <pentry minsize="1" maxsize="4">
+          <register name="r10"/>
+        </pentry>
+      </output>
+      
+      <unaffected>
+        <register name="sp"/>
```

Comment:
You still need to add the registers that should be unaffected in the RH850 spec: r20-31
See https://www.renesas.com/us/en/doc/products/tool/doc/015/r20ut3516ej0107-ccrh.pdf section 9.1. As far as I can tell r3-r5 (sp, gp and tp) should not be listed as unaffected.